README.md

```diff
@@ -116,7 +116,11 @@ kubectl create -f bundle.yaml
 To run the Operator outside of a cluster:
 
 ```sh
-make
+kubectl create -f example/prometheus-operator-crd-full
+```
+
+```sh
```

Comment:
The command at L111 would already create/update the CRDs and also apply the needed RBAC objects.

```suggestion
```

Reply:
How about write this section as .. 

```
kubectl delete -f bundle.yaml   # to remove already running PO 

kubectl create -f example/prometheus-operator-crd-full
make operator
./scripts/run-external.sh
```

Reply:
I know this is beyond the scope of this change but .. since user has to run `make operator` every single tinme before running the script, how about we add that to the script itself? 
In fact all of these can be incorportated  into the script so that to run, PO, all someone has to do is to run the script.

Reply:
Here is an example -  https://github.com/prometheus-operator/prometheus-operator/pull/5415 which I am using locally to run operator out of cluster.

Reply:
Thinking more about it, I'd move all developer instructions to CONTRIBUTING.md and have only user-oriented content in the README.